pkgs/development/libraries/glibc/default.nix

```diff
@@ -4,6 +4,7 @@
 , withGd ? false
 , withLibcrypt? false
 , buildPackages
+, enableCopyLibGccHack ? !stdenv.enableGccExternalBootstrapForStdenv or false
```

Comment:
> Making these things optional makes it very hard to reason about the system various configurations.

It's an artifact of my abandoned attempt to break this PR up into separate PRs that could pass CI independently in order to be reviewed.

I now acknowledge that it was a mistake to attempt that.

I've made this a local variable instead of a parameter.  It will be dropped as deadcode if/when we drop support for internal bootstrap.